<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasureOn6DiseaseClass</class>
 <widget class="QMainWindow" name="MeasureOn6DiseaseClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MeasureOn6Disease</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QVTKOpenGLNativeWidget" name="pvtkQtWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>930</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="actionReset"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit_2"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>Operation</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="buttonAMD">
         <property name="text">
          <string>A</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonDME">
         <property name="text">
          <string>B</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonDRY">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>C</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonNM">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>D</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonPCV">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>E</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonPM">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>F</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="buttonSave">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonResetColor">
         <property name="text">
          <string>ResetColor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonResetWindow">
         <property name="text">
          <string>ResetWindow</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonClose">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>qvtkopenglnativewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MeasureOn6Disease.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonAMD</sender>
   <signal>clicked(bool)</signal>
   <receiver>MeasureOn6DiseaseClass</receiver>
   <slot>OnClickAMD()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonDME</sender>
   <signal>clicked(bool)</signal>
   <receiver>MeasureOn6DiseaseClass</receiver>
   <slot>OnClickDME()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonDRY</sender>
   <signal>clicked(bool)</signal>
   <receiver>MeasureOn6DiseaseClass</receiver>
   <slot>OnClickDRY()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonNM</sender>
   <signal>clicked(bool)</signal>
   <receiver>MeasureOn6DiseaseClass</receiver>
   <slot>OnClickNM()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonPCV</sender>
   <signal>clicked(bool)</signal>
   <receiver>MeasureOn6DiseaseClass</receiver>
   <slot>OnClickPCV()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonPM</sender>
   <signal>clicked(bool)</signal>
   <receiver>MeasureOn6DiseaseClass</receiver>
   <slot>OnClickPM()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonResetColor</sender>
   <signal>clicked()</signal>
   <receiver>MeasureOn6DiseaseClass</receiver>
   <slot>OnClickCheckResetColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonResetWindow</sender>
   <signal>clicked()</signal>
   <receiver>MeasureOn6DiseaseClass</receiver>
   <slot>OnFileReset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonClose</sender>
   <signal>clicked()</signal>
   <receiver>MeasureOn6DiseaseClass</receiver>
   <slot>OnFileClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSave</sender>
   <signal>clicked()</signal>
   <receiver>MeasureOn6DiseaseClass</receiver>
   <slot>OnFileSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>90</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnClickAMD()</slot>
  <slot>OnClickDME()</slot>
  <slot>OnClickDRY()</slot>
  <slot>OnClickNM()</slot>
  <slot>OnClickPCV()</slot>
  <slot>OnClickPM()</slot>
  <slot>OnClickCheckResetColor()</slot>
  <slot>OnFileReset()</slot>
  <slot>OnFileClose()</slot>
  <slot>OnFileSave()</slot>
 </slots>
</ui>
